hummingbot/connector/exchange/ndax/ndax_user_stream_tracker.py

```diff
@@ -39,7 +43,9 @@ def data_source(self) -> UserStreamTrackerDataSource:
         :return: UserStreamTrackerDataSource
         """
         if not self._data_source:
-            self._data_source = NdaxAPIUserStreamDataSource(auth_assistant=self._auth_assistant, domain=self._domain)
+            self._data_source = NdaxAPIUserStreamDataSource(
+                self._throttler, auth_assistant=self._auth_assistant, domain=self._domain
```

Comment:
Just a code styling issue. Do include the expected parameter since it is available.
i.e. `ds = NdaxAPIUserStreamDataSource(throttler=self._throttler, ...)`

Reply:
Changed it to match the style of the existing code. However, while I am very much pro passing arguments by key-value when they add additional information (e.g. `order = self.place_buy_order(price=30)`, I think in cases like these, it adds verbosity without necessarily increasing the information